src/nodejs/CMakeLists.txt

```diff
@@ -33,8 +33,8 @@ endforeach(binary)
 
 # For mason-enabled builds we copy over tbb's shared objects for packaging.
 # TODO: consider using statically linked tbb library (for node_osrm only!)
```

Comment:
Btw I think it can be possible to try(we already doing that for Windows builds). @mjjbell do you see some downsides of it?

Reply:
I don't have enough context on how the Node packaging works, but it doesn't look like we'd be losing much as we're not actually getting the benefit of a shared lib. If anything, statically linking might reduce the package size.

Reply:
Okay, will try to check how it will work :+1: